image/remote_handler.go

```diff
@@ -9,18 +11,31 @@ import (
 const RemoteKind = "remote"
 
 type RemoteHandler struct {
-	keychain authn.Keychain
+	keychain           authn.Keychain
+	insecureRegistries []string
 }
 
 func (h *RemoteHandler) InitImage(imageRef string) (imgutil.Image, error) {
 	if imageRef == "" {
 		return nil, nil
 	}
 
+	options := []remote.ImageOption{
+		remote.FromBaseImage(imageRef),
+	}
+
+	if len(h.insecureRegistries) > 0 {
+		for _, insecureRegistry := range h.insecureRegistries {
+			if strings.HasPrefix(imageRef, insecureRegistry) {
+				options = append(options, remote.WithRegistrySetting(insecureRegistry, true, true))
```

Comment:
The RFC says `The second parameter specify whether the registry is insecure. The third parameter can be always false.` ...which, it's a little confusing - why have a third parameter if it's always false? According to imgutil the third parameter disables SSL certificate validation. I wonder if we should try exposing it to the end user somehow...

Reply:
Yeah, my thought here was that if we are using `WithRegistrySetting` it means we need to skip any SSL validation so I left that as a true.
What do you think? I agree that if we are considering of using always a specific value we should change the signature on imgutil